<ui version="4.0" >
 <class>CheckCDWidget</class>
 <widget class="QWidget" name="CheckCDWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>131</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="pix" >
       <property name="text" >
        <string>pix</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabel2" >
       <property name="text" >
        <string>You can check the integrity of the packages in installation CD. Although this is an optional step, we encourage you to do it to avoid any errors in installation.</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
       <property name="margin" >
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>21</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="checkButton" >
         <property name="text" >
          <string>Check CD Integrity</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>21</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="checkLabel" >
       <property name="minimumSize" >
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="margin" >
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>131</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
